packages/core/src/asciidoctor-core-api.js

```diff
@@ -550,6 +550,25 @@ AbstractBlock.prototype.convert = function () {
  * that match the specified selector (context, style, id, and/or role).
  * If a function block is given, it's used as an additional filter.
  * If no selector or function block is supplied, all block-level nodes in the tree are returned.
+ * Valid context names include: 
```

Comment:
Could you please update the list with https://docs.asciidoctor.org/asciidoc/latest/blocks/#summary-of-built-in-contexts
And could you also please add a link to this documentation page?